scholia/app/templates/venue.html

```diff
@@ -39,7 +39,7 @@
 
  prolificAuthorsSparql = `
 # Prolific authors for a specific journal
-SELECT ?count ?author ?authorLabel ?orcid
+SELECT ?count ?author ?authorLabel ?ORCID
```

Comment:
Should I change back the variable name?

Do I understand correctly, we can upper case the table column headers later?